src/scripts/site.js

```diff
@@ -309,6 +311,10 @@ var AppInfo = {};
         return "bower_components";
     }
 
+    function getComponentsPath() {
```

Comment:
why is this needed?

Reply:
I was just matching the `getBowerPath()` method above, we might want to eventually remove this but right now the path is set in multiple locations.